### ISSUE TITLE
Fix akka-bench and update plugin

### ISSUE THREAD

**Issue by ktoso** [akka/akka#19481]
(streams imported benchmarks break compilation under latest JMH)

Also update plugin to the latest

**Comment by agolubev** [comment]
I'm having following log on jmh:run 
```
akka-bench-jmh > jmh:run
[info] Compiling 1 Scala source to /Users/alexandergolubev/Projects/akka/akka-bench-jmh/target/classes...
Processing 184 classes from /Users/alexandergolubev/Projects/akka/akka-bench-jmh/target/classes with "reflection" generator
Writing out Java source to /Users/alexandergolubev/Projects/akka/akka-bench-jmh/target/src_managed/jmh and resources to /Users/alexandergolubev/Projects/akka/akka-bench-jmh/target/resource_managed/jmh
@Param annotation is not acceptable on final fields.
   [org.openjdk.jmh.generators.reflection.RFFieldInfo@2c17ecad]

@Param annotation is not acceptable on final fields.
   [org.openjdk.jmh.generators.reflection.RFFieldInfo@4a31409b]

@Param annotation is not acceptable on final fields.
   [org.openjdk.jmh.generators.reflection.RFFieldInfo@684866e2]

@Param annotation is not acceptable on final fields.
   [org.openjdk.jmh.generators.reflection.RFFieldInfo@47208be4]

@Param annotation is not acceptable on final fields.
   [org.openjdk.jmh.generators.reflection.RFFieldInfo@54417529]

@Param annotation is not acceptable on final fields.
   [org.openjdk.jmh.generators.reflection.RFFieldInfo@14b4d2a2]
```

have this fixed in PR #19701 about Substreams Performance

**Comment by ktoso** [comment]
Yeap, that's the only problem that needs to be addressed here, thanks a lot for solving it in your PR over there.
